src/components/navigation/questionNavigation/QuestionStep.tsx

```diff
@@ -0,0 +1,170 @@
+import { Box, SxProps, Tooltip, Typography } from '@mui/material';
+
+import { buildQuestionStepCy } from '../../../config/selectors';
+import theme from '../../../layout/theme';
+import QuestionStepIcon from './QuestionStepIcon';
+import QuestionTitleStepper from './QuestionTitleStepper';
+
+const DEFAULT_SIZE = 10;
+const CONTAINER_SIZE = 27;
+const SELECTED_SIZE = 23;
+const DEFAULT_SHADOW = '0px 1px 1px rgb(0, 0, 0, 0.25)';
+const DEFAULT_BORDER_WIDTH = '2px';
+const DEFAULT_BORDER_STYLE = 'solid';
+const BORDER_RADIUS = '50%';
+export const INCORRECT_BORDER_COLOR = theme.palette.error.dark;
+export const ATTEMPTS_BORDER_COLOR = theme.palette.error.dark;
+export const CORRECT_BORDER_COLOR = theme.palette.success.dark;
+
+const DEFAULT_SX = {
+  borderWidth: DEFAULT_BORDER_WIDTH,
+  borderRadius: BORDER_RADIUS,
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  cursor: 'pointer',
+
+  transition: 'width 0.5s, height 0.5s, background 0.2s',
+
+  '&:hover': {
+    height: SELECTED_SIZE,
+    width: SELECTED_SIZE,
+    boxShadow: 'none',
+    borderStyle: DEFAULT_BORDER_STYLE,
+  },
+};
+
+export enum QuestionStepStyleKeys {
+  DEFAULT = 'default',
+  CORRECT = 'correct',
+  INCORRECT = 'incorrect',
+  REMAIN_ATTEMPTS = 'remain_attempts',
+}
```

Comment:
I'm wondering if it's shouldn't go in a separate file `types.ts` (I guess it's okay in folder `/questionNavigation`), it's up to you.